wordpress-mobile/WordPress-Android
# Do not display password for self-hosted sites

Fixes #9899 

This PR prevents an attacker from viewing a password for self-hosted sites.

![update-password](https://user-images.githubusercontent.com/2261188/58640810-25419280-82fa-11e9-8297-e5f15f216922.gif)


To test:
- Make sure the self-hosted site password is not visible ->My Sites > Settings > Tap on Password -> Notice the Edit text is empty
- Make sure that you can't set the password to an empty string
- Make sure a toast is displayed when you update the password
- Turn on airplane mode and notice all the preferences get disabled
- Make sure the password actually gets updated -> change it to a wrong password and go to My Site -> Blog Posts -> notice an error message in the LogCat

Update release notes:

- The change is too minor

**Note: Needs copy review -> the strings aren't final.**
